tsfile/src/main/java/org/apache/iotdb/tsfile/utils/ReadWriteIOUtils.java

```diff
@@ -82,6 +83,19 @@ public static boolean readBool(ByteBuffer buffer) {
     return a == 1;
   }
 
+  /**
+   * read a byte from inputStream.
+   */
+  public static byte readByte(InputStream inputStream) throws IOException {
```

Comment:
hm.. this method is overdesigned..

If you read `InputStream.read(byte[])` method, you will find it actually just call `byte c = (byte)read()`;
So, just `return (byte) read()` is ok.

Reply:
OK, thanks for your guidance